es/13_0_Disenando_Bitcoin_Scripts_Reales.md

```diff
@@ -1,4 +1,4 @@
-# Capítulo 13: Diseño de scripts de Bitcoin reales
+# Capítulo 13: Diseñando Bitcoin Scripts Reales
```

Comment:
Oh, I forget about Capital letters:
`# Capítulo 13: Diseñando Bitcoin Scripts reales` is better, isn't it?